rollup.config.js

```diff
@@ -15,12 +15,14 @@ export default [
     output: {
       file: pkg.main,
       format: 'cjs',
+      sourcemap: true,
     },
     external,
     plugins: [
       babel({
         runtimeHelpers: true,
         plugins: ['@babel/transform-runtime'],
+        sourceMap: true,
```

Comment:
I think `sourceMaps` is the recommended option name.

Reply:
Whoops, fixed. Between `sourcemap`, `sourceMap`, and `sourceMaps` it's fun remembering which build tool likes which capitalization.

Reply:
No kidding.